src/webextensionadapter.cpp

```diff
@@ -153,7 +153,7 @@ QJsonObject WebExtensionAdapter::serializeStatus() {
   {
     Controller::State state = vpn->controller()->state();
     if (state == Controller::StateOnPartial) {

```

Comment:
I think you can just remove the lines :)

Reply:
😵‍💫 embarrassing